game-core/src/main/java/games/strategy/engine/data/gameparser/XmlGameElementMapper.java

```diff
@@ -165,16 +167,11 @@
       return Optional.empty();
     }
 
-    if (className.startsWith("games.strategy.twoIfBySea.delegate.")) {
-      ClientLogger.logQuietly("Use of twoIfBySea delegates is discouraged "
-          + "and will be removed in a future version of TripleA.");
-    }
```

Comment:
I don't disagree that this is somewhat important information, but it's not appropriate here:
- as a quiet logging, it's ignored.
- not good that this is in code, after the fact like this. Should be on PoS.xml, forum post or a github issue. We should also probably deprecate the delegate.
- Beyond that, there are some problems with the text. "Discouraged" is meaningless, if someone finds a good use then we're going to wind up support it.. Otherwise, what does 'discouraged' actually mean?

Reply:
Ok, I get that.
I originally added that message there to sort of promote the short notation feature of delegate names, because I noticed there were some name collusions and didn't want map makers to confuse one for the other.
Turned out the twoIfBySea are only use by a handful of experimental maps that are no longer being maintained, so we probably want to migrate them and remove those delegates